test/ref/multinomial.cpp

```diff
@@ -78,5 +170,6 @@ TEST_CASE(multinomial_test)
     std::transform(res_dist.begin(), res_dist.end(), res_norm.begin(), [&](auto n) {
         return static_cast<double>(n) / res_dist_sum;
     });
+    // The given test tolerance is about 10x the typical error
```

Comment:
Wouldn't this test tolerance be `1e6 / 80` of the default tolerance? The default multiplier we have on the epsilon is 80.

Reply:
This is with 100,000 samples, so that's not necessarily such a large tolerance.

Reply:
This test has been replaced with verify_range_with_tolerance()